models/stg_recharge__payment_source.sql

```diff
@@ -1,5 +1,5 @@
-{{ config(enabled=var('recharge__payment_source_enabled', True)) }}
-with base as (
+{{ config(enabled=var('recharge__payment_source_enabled', True)) }} 
+with base as ( 
```

Comment:
Just a small note that this will likely be changed to payment_method.

Reply:
I updated the name and can use identifiers to get around the table name issue, however I'm noticing the columns have changed. I can make mock data in our seed, but then I can't run any real data. Any suggestions how to get around/reconcile the column issue?

Reply:
As discussed internally we will avoid the column differences for the time being. We can account for those once the ERD changes are reflected in our data set.